prow/cmd/webhook-server/helpers.go

```diff
@@ -131,3 +205,153 @@ func isCertValid(cert string) error {
 	}
 	return nil
 }
+
+func getGCPSecrets(client ClientInterface, ctx context.Context, expiry int, dns []string) (string, string, string, error) {
```

Comment:
these functions could work with GCP or localFS, please remove `GCP` from the names of the functions